src/pages/signin/PasswordForm.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import {Text, TextInput, View} from 'react-native';
+import PropTypes from 'prop-types';
+import {withOnyx} from 'react-native-onyx';
+import {withRouter} from '../../libs/Router';
+import styles from '../../styles/styles';
+import SubmitButton from './SubmitButton';
+import themeColors from '../../styles/themes/default';
+import {signIn} from '../../libs/actions/Session';
+import compose from '../../libs/compose';
+import ONYXKEYS from '../../ONYXKEYS';
+
+const propTypes = {
+    // These are from withRouter
+    // eslint-disable-next-line react/forbid-prop-types
+    match: PropTypes.object.isRequired,
+
+    /* Onyx Props */
+
+    // The details about the account that the user is signing in with
+    account: PropTypes.shape({
+        // Whether or not the account already exists
+        accountExists: PropTypes.bool,
+
+        // Whether or not there have been chat reports shared with this user
+        canAccessExpensifyCash: PropTypes.bool,
+
+        // Whether or not two factor authentication is required
+        requiresTwoFactorAuth: PropTypes.bool,
+    }),
+};
+
+const defaultProps = {
+    account: {},
+};
+
+class PasswordForm extends React.Component {
```

Comment:
NAB, it wouldn't hurt to add brief descriptions to each of these views to explain in which situations they are used.